adagrams/game.py

```diff
@@ -1,11 +1,121 @@
+#from tests.test_wave_01 import LETTER_POOL
+import random
+import copy
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_DICT = {
+    1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+    2: ['D', 'G'],
+    3: ['B', 'C', 'M', 'P'],
+    4: ['F', 'H', 'V', 'W', 'Y'],
+    5: ['K'],
+    8: ['J', 'X'],
+    10: ['Q', 'Z'],
+}
+
 def draw_letters():
-    pass
+    # generate list containing all letters
+    # in corresponding frequency
+    all_letters = []
+    for letter,frequency in LETTER_POOL.items():
+        for i in range(frequency):
+            all_letters.append(letter)
```

Comment:
👍 Nice way to build up the list of all "tiles". Since we're not using `i` for anything here, it's common to use `_` as a variable name instead. It communicates to anyone reading the code that we're not using this variable for anything, but a variable was required for the syntax.

Alternatively, rather than doing the inner loop ourself, we could instead extend the letter pool with a string made by repeating the current letter like this:
```python
    all_letters = []
    for letter, frequency in LETTER_POOL.items():
            all_letters += letter * frequency
```

The result would be the same.